components/shared/Play/Play.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { useState } from "react";
+import Icon from "../Icon/Icon";
+import "./Play.module.scss";
+
+interface PlayProps {
+  onClick: () => void;
+}
+
+const Play: React.FC<PlayProps> = ({ onClick }) => {
+  const [isPlaying, setIsPlaying] = useState(false);
+
+  const handleTogglePlay = () => {
+    setIsPlaying(!isPlaying);
```

Comment:
Well, this code won't work correctly. Coz, react takes some time before update state. So, it can happen situation, when you still get an old state value. I'd recommend use callback, to get previous state 🙂
```suggestion
    setIsPlaying(prevState => !prevState);
```